### ISSUE TITLE
Error on opening this plugin (/public/emotion status=404)

### ISSUE THREAD

**Issue by jayaraj** [marcusolsson/grafana-calendar-panel#31]
I get these errors while loading this plugin.

```
INFO[01-17|08:44:07] Request Completed                        logger=context userId=1 orgId=1 uname=admin method=GET path=/api/plugins/marcusolsson-calendar-panel/markdown/help status=404 remote_addr=[::1] time_ms=11 size=68 referer="http://localhost:3000/plugins/marcusolsson-calendar-panel?page=overview"
INFO[01-17|08:44:11] Downloaded marcusolsson-calendar-panel v0.5.0 zip successfully logger=plugin.installer
INFO[01-17|08:44:11] Plugin registered                        logger=plugin.manager pluginId=marcusolsson-calendar-panel
INFO[01-17|08:44:11] Request Completed                        logger=context userId=1 orgId=1 uname=admin method=GET path=/public/emotion status=404 remote_addr=[::1] time_ms=21 size=37403 referer="http://localhost:3000/plugins/marcusolsson-calendar-panel?page=overview"
```
![image](https://user-images.githubusercontent.com/364636/150759080-21a2d72e-86a0-4944-919d-b02547816c70.png)

**Comment by marcusolsson** [comment]
Should be fixed in 0.6.0 and 0.7.0. Let me know if not, and I'll reopen the issue!

**Comment by jayaraj** [comment]
I am this issue again on 0.7.0
